src/transpiler.ts

```diff
@@ -403,6 +405,44 @@ function transformUnaryAndBinaryOperationsToFunctionCalls(program: es.Program) {
   })
 }
 
+function transformPropertyAssignment(program: es.Program) {
+  simple(program, {
+    AssignmentExpression(node) {
+      const assignmentExpression = node as es.AssignmentExpression
+      if (assignmentExpression.left.type === 'MemberExpression') {
+        const { object, property, computed, loc } = assignmentExpression.left
+        const { line, column } = loc!.start
+        node = node as es.CallExpression
```

Comment:
Is it possible to use create.mutateToCallExpression here?

Reply:
Yes. Seems like I made it because it was horrible to change something to callExpression, but forgot all about it.